source/features/action-used-by-link.tsx

```diff
@@ -4,10 +4,10 @@ import {SearchIcon} from '@primer/octicons-react';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import selectHas from '../helpers/select-has';
 
 function init(): void {
-	const actionRepo = select('aside .octicon-repo')!
-		.closest('a')!
+	const actionRepo = selectHas('a:has(aside .octicon-repo)')!
```

Comment:
These selectors are real, valid selectors and can be tested natively in Safari. In the future we'll be able to just replace `selectHas` with `select`

Reply:
I'm tempted to `#enable-experimental-web-platform-features` now

Or maybe install Chrome Dev which just added `:has()` support in 105. But it is scheduled to release on Aug 30 according to [Chromium Dash](https://chromiumdash.appspot.com/schedule), so I'll just wait for two months 😆

Reply:
Firefox will delay this a while longer. No work in progress at all. https://bugzilla.mozilla.org/show_bug.cgi?id=418039

I think we can’t try the native implementation until we’re ready to ship or else we need to test each selector for both native and polyfilled version. Keep in mind that the polyfill is very limited so we can’t trust it to match the native implementation yet.